src/components/ManageAccountsDrawer/components/ImportAccounts.tsx

```diff
@@ -45,18 +49,28 @@ type TableRowProps = {
   accountId: AccountId
   accountNumber: number
   asset: Asset
-  toggleAccountId: (accountId: AccountId) => void
+  onAccountIdActiveChange: (accountId: AccountId, isActive: boolean) => void
 }
 
 const disabledProp = { opacity: 0.5, cursor: 'not-allowed', userSelect: 'none' }
 
 const TableRow = forwardRef<TableRowProps, 'div'>(
-  ({ asset, accountId, accountNumber, toggleAccountId }, ref) => {
+  ({ asset, accountId, accountNumber, onAccountIdActiveChange }, ref) => {
     const translate = useTranslate()
-    const handleChange = useCallback(() => toggleAccountId(accountId), [accountId, toggleAccountId])
     const accountLabel = useMemo(() => accountIdToLabel(accountId), [accountId])
     const balanceFilter = useMemo(() => ({ assetId: asset.assetId, accountId }), [asset, accountId])
+    const isAccountEnabledFilter = useMemo(() => ({ accountId }), [accountId])
+    const isAccountEnabledInRedux = useAppSelector(state =>
```

Comment:
Don't think we need the `redux` terminology if we  have two distinct `enabled`, and `toggled`/`active` terminologies
```suggestion
    const isAccountEnabled = useAppSelector(state =>
```

Reply:
good point, ill fix this in a follow-up as it'll introduce conflicts with the next PR